### PR TITLE
updated Definitions for Raven-JS

### PR DESCRIPTION
**Pull request by LucaVazz**
- [X] Make your PR against the `master` branch.
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [X] Run `tsc` without errors.

- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/getsentry/raven-js/pull/827 and https://github.com/getsentry/raven-js/pull/820>>

### CODE REVIEW

**Comment by LucaVazz** [comment]
Could someone please help me with the Result of the Travis CI Build?

>  Typing Publish Report Summary
> Started at Thu, 12 Jan 2017 23:56:08 GMT
> Found 2814 typings folders in /home/travis/build/DefinitelyTyped/DefinitelyTyped
> Error: In raven-js: At 2:50 : Expected one of
>	/(, )|(,?\r?\n\/\/\s\s+)/
>	/\r?\n\/\/ Definitions by: /
>    at Object.parseHeaderOrFail (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/src/lib/header.ts:35:9)
>    at /home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/src/lib/definition-parser.ts:89:3
>    at next (native)
>    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/types-publisher/bin/lib/definition-parser.js:4:58)
>npm ERR! Test failed.  See above for more details.

I don't understand what the log is trying to say...

File: raven-js/index.d.ts [hunk 1]
```diff
@@ -1,212 +1,305 @@
 // Type definitions for Raven.js
 // Project: https://github.com/getsentry/raven-js
-// Definitions by: Santi Albo <https://github.com/santialbo/>, Benjamin Pannell <http://github.com/spartan563>, Gary Blackwood <http://github.com/Garee>, Rich Rout <http://github.com/richrout>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Original Definitions by: Santi Albo <https://github.com/santialbo>, Benjamin Pannell 
+//   <http://github.com/spartan563>; DefinitelyTyped
+// Updated by: Ben Vinegar <https://github.com/benvinegar>, Ilya Pirogov
```
Comment (Garee) [inline raven-js/index.d.ts]: There are two names missing and the removal of `Definitions by:` and `Definitions:` will cause the tests to fail.

Why did you add all the other names?

Comment (LucaVazz) [inline raven-js/index.d.ts]: I added the other names because they contributed to the version of the file over at the [Raven-JS](https://github.com/LucaVazz/raven-js/blob/master/typescript/raven.d.ts)-Project's Repo.

Comment (Garee) [inline raven-js/index.d.ts]: Fair enough, didn't know about that. The error message is complaining about the missing `Definitions by:`.

Comment (LucaVazz) [inline raven-js/index.d.ts]: Should I change it to
```
// Definitions by: Santi Albo <https://github.com/santialbo/>, Benjamin Pannell <http://github.com/spartan563>, Gary Blackwood <http://github.com/Garee>, Rich Rout <http://github.com/richrout>, Ben Vinegar <https://github.com/benvinegar>, Ilya Pirogov <https://github.com/ilya-pirogov>, Eli White <https://github.com/TheSavior>, David Cramer <https://github.com/dcramer>, Connor Peet <https://github.com/connor4312>, comaz <https://github.com/combmag>, Luca Vazzano <https://github.com/LucaVazz>
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
or rather drop the additional names?

Comment (LucaVazz) [inline raven-js/index.d.ts]: Oh, didn't see your comment. okay, I will add the line as `Definitions by:`

**Comment by mhegazy** [comment]
@Garee any more comments?

**Comment by bowdenk7** [comment]
No response from other authors for over 5 days. Merging.
